### PR TITLE
Parse graph legend mnrva 813

### PR DESCRIPTION
**Pull request by pras7430**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-813

## Description:

Added a Regex to handle class name(replaced special char with _) legend text and line.

## Screenshots:
(if applicable)

### CODE REVIEW

File: docs/assets/scripts/hedwig-main.umd.js [hunk 1]
```diff
@@ -6540,6 +6540,7 @@
     stylesheet: stylesheet
   });
 
+  const spclCharRegx = /[&\/\\#, +()$~%.'":*?<>{}]/g;
```
Comment (lovedeep92) [inline docs/assets/scripts/hedwig-main.umd.js:6543]: Hey, can you please remove this file from commit as it is generated after we run build. so no need to add this!

File: src/line-graph.js [hunk 2]
```diff
@@ -345,22 +354,22 @@ export class LineGraph extends HTMLElement {
...
         if (d.group) {
-          return d.group.split('/')[d.group.split('/').length-1];
+          return d.group;
         } else {
           return this.dataset.field
         }
       })
       // Legend text Click
       .on('click', function (d,i) {
+        let cssCls= `${d.group}${i}`.replace(spclCharRegx,'_');
```
Comment (lovedeep92) [inline src/line-graph.js:364]: 👍

**Review by lovedeep92 — APPROVED** [review]

File: docs/assets/scripts/hedwig-main.umd.js [hunk 3]
```diff
@@ -6567,8 +6568,11 @@
       this.attachShadow({
         mode: 'open'
       });
-      this.shadowRoot.appendChild(svg);
-      this.renderGraph(this.parseData(data), svg);
+
```
Comment (lovedeep92) [inline docs/assets/scripts/hedwig-main.umd.js:6571]: Is this necessary to add the same code in this file?

**Review by pras7430 — COMMENTED** [review]
auto-generated

File: docs/assets/scripts/hedwig-main.umd.js (hunk 3 above)
Comment (pras7430) [reply docs/assets/scripts/hedwig-main.umd.js:6571]: auto-generated

**Review by FarajiA — APPROVED** [review]

File: docs/assets/scripts/hedwig-main.umd.js (hunk 3 above)
Comment (lovedeep92) [reply docs/assets/scripts/hedwig-main.umd.js:6571]: yes .. so I don't think it needs to be committed from local!

**Pull request merged**